DataDefinitions/FactionRecord.cs

```diff
@@ -0,0 +1,182 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.Serialization;
+
+namespace EddiDataDefinitions
+{
+    /// <summary>
+    /// Record defines the claims, fines and bounties associated with a faction, along with some additional data
+    /// </summary>
+    public class FactionRecord : INotifyPropertyChanged
```

Comment:
This is a nitpick, but I find the name `FactionRecord` to not be very intuitive as far as understanding what this does and how it differs from a `Faction` object. Consider `FactionLedger` as an alternate?

Reply:
This is a nitpick. It's a faction 'criminal record'.

Reply:
Then perhaps `CrimeRecord`? `FactionRecord` makes me think that it might contain information about the pilots missions history and/or reputation.